### DISCUSSION TITLE
[BANDCHAIN] Not possible to approve transaction on Ledger

### DISCUSSION THREAD

**Discussion by federik95** [ping-pub/explorer#493] [Q&A]
Hello, some times ago I staked some Band by Ping.pub. It was connect with Keplr (itself connected with the Ledger). 
Now I'm trying to connect the wallet to Ping.pub but: 

1) If I connect with Keplr and try to, for example, withdraw staking rewards, I can't approve the transaction with the error: Incompatible signing requested: Error: SIGN_MODE_DIRECT can’t be signed on Ledger. Contact the web app provider to fix this issue.

2) If I connect directly with Ledger I receive instantly the error: Error: Ledger Native Error: Data is invalid.

What can I do to have access to my wallet?

Thank you

**Comment by liangping** [comment]
will try to use sign amino
